### ISSUE TITLE
Will Nusic be updated? 

### ISSUE THREAD

**Issue by uniquePWD** [schnatterer/nusic#27]
Nusic is one of the oldest apps still on my phone. I'm wondering if it's likely to see an update or if I should just pour out a little liquor and move on from it.

**Comment by schnatterer** [comment]
@pwd-github 

TL;DR I'm not working on nusic, but still using it. If you have found a contemporary app with the same functionality please share.

Unfortunately, I don't have any free time to work on my private projects at the moment. Also upgrading android apps to newer API versions (android major versions) is a major PITA, especially for non-commercial app devs. You spent most of your time on refactorings just to keep track with the latest API changes in android and gradle. No time left for new features...
nusic is currently on API Level 24, Android 12 is 31. It might be easier to completely rewrite the Android parts of nusic. But who would have time for that?
Google will provably kick nusic from the play store in November, because it does not support newer API levels. It will continue to work, though and still be available on GitHub and F-Droid.

The basic functionality of nusic is still there on Android 12. UX could be better, but I'm not aware of alternatives. So I will continue using it.
